platform/golden/bullseye.pkr.hcl

```diff
@@ -1,59 +1,107 @@
-variables {
-  headless          = true
-  iso_checksum      = "224cd98011b9184e49f858a46096c6ff4894adff8945ce89b194541afdfd93b73b4666b0705234bd4dff42c0a914fdb6037dd0982efb5813e8a553d8e92e6f51"
-  iso_checksum_type = "sha512"
-  iso_url           = "https://cdimage.debian.org/debian-cd/11.6.0/amd64/iso-cd/debian-11.6.0-amd64-netinst.iso"
+variable "config_file" {
+  type    = string
+  default = "debian-11-preseed.cfg"
+}
+
+variable "config_dir" {
+  type    = string
+  default = "cfg"
+}
+
+variable "cpus" {
+  type    = string
+  default = "4"
+}
+
+variable "disk_size" {
+  type    = string
+  default = "4096"
+}
+
+variable "memory" {
+  type    = string
+  default = "16384"
+}
+
+variable "headless" {
+  type    = string
+  default = "true"
+}
+
+variable "iso_checksum" {
+  type    = string
+  default = "eb3f96fd607e4b67e80f4fc15670feb7d9db5be50f4ca8d0bf07008cb025766b"
+}
+
+variable "iso_checksum_type" {
+  type    = string
+  default = "sha256"
+}
+
+variable "iso_url" {
+  type    = string
+  default = "https://cdimage.debian.org/debian-cd/11.7.0/amd64/iso-cd/debian-11.7.0-amd64-netinst.iso"
+}
+
+variable "image_name" {
+  type    = string
+  default = "golden"
+}
+
+variable "ssh_password" {
+  type    = string
+  default = "packer"
```

Comment:
defaults should not be permitted in passwords.

Reply:
I'll remove the default, and make it a variable created in the run.sh script.